README.md

```diff
@@ -1,10 +1,35 @@
-# @crowdstrike/ember-toucan-core
+# Toucan Core
+
+<p align="center">
+  <a href="https://github.com/crowdstrike/ember-toucan-core">
+    <img src="https://github.com/CrowdStrike/ember-toucan-core/tree/main/.github/cs-logo.png?raw=true" alt="CrowdStrike logo" width="300" />
```

Comment:
**NOTE:** If you [view this file rendered](https://github.com/CrowdStrike/ember-toucan-core/blob/b3a2a27229cda2c2c8dc78f1366d544ed3b59513/README.md), you will notice the image is not displayed.  That's because the image is not yet on `main`.  Once this merges, it should be good to go 👍.  A bit of a chicken/egg problem.

To test this out you could manually edit the `<img` `src` attribute to `src="https://github.com/CrowdStrike/ember-toucan-core/blob/update-readme/.github/cs-logo.png?raw=true"` and you should see:

<img width="1016" alt="Screenshot 2023-06-16 at 8 44 21 AM" src="https://github.com/CrowdStrike/ember-toucan-core/assets/8069555/fc467667-aa7e-40f5-8e75-0fcd9c012d5e">